src/msgwindow.c

```diff
@@ -83,6 +83,11 @@ enum
 };
 
 
+static GdkColor color_error = {0, 65535, 0, 0};
+static GdkColor color_context = {0, 65535 / 2, 0, 0};
+static GdkColor color_message = {0, 0, 0, 0xD000};
```

Comment:
OK I see it comes from the old code.  Well, as you want then, but could be the occasion to make that less odd :)